scripts/mixins/industrialcraft2.zs

```diff
@@ -0,0 +1,424 @@
+#loader mixin
```

Comment:
I see that `bansoukou/industrialcraft-2-2.8.222-ex112/ic2/core/item/ItemBatteryChargeHotbar.class` was removed, but cant see any tweaks of this class in current file.
Is changes was accidentally missed or implemented other way?